lib/iex/lib/iex/evaluator.ex

```diff
@@ -83,20 +83,38 @@ defmodule IEx.Evaluator do
     file = Keyword.get(opts, :file, "nofile")
     line = Keyword.get(opts, :line, 1)
     column = Keyword.get(opts, :column, 1)
+    last_op = Keyword.get(opts, :last_op, :none)
```

Comment:
This is a state that is specific to the evaluator, so I don't think it should surface all the way up to the IEx server. The trick here is to change the evaluator state. The first clause should be:

  def parse(input, opts, ""), do: parse(input, opts, {"", :none})

And now always return a tuple as the evaluation state, where the second element is the last operation.